.github/workflows/CI.yml

```diff
@@ -0,0 +1,80 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This workflow will install a prebuilt Ruby version, install dependencies, and
+# run tests and linters.
+name: "Generate sample app and run tests"
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+jobs:
+  test_html:
+    name: 🧪 Run HTML Tests
+    runs-on: ubuntu-latest
+    services:
+      redis:
```

Comment:
I think we can remove redis, since we are not testing the optional properties... The basic generator doesn't require redis